packages/ui-extensions/src/surfaces/admin/shouldRender.ts

```diff
@@ -0,0 +1,20 @@
+import {createDataRegistrationFunction} from '../../utilities/dataRegistration';
+
+import type {ExtensionTargets} from './should-render-targets';
+
+interface ShopifyDataGlobal {
```

Comment:
Let's remove these since we will no longer expose the `dataExtension` global directly. We expect people to only use `shouldRender` for now